config/default.yaml

```diff
@@ -97,20 +97,33 @@ storage:
 
 object_storage:
   enabled: false
-  # Maximum amount of MiB to upload in one request to object storage
-  max_upload_part: 2048
-  # Will always use https with default URL generation (see below)
+
+  # Without protocol, will default to HTTPS
   endpoint: 's3.amazonaws.com'
+
+  region: 'us-east-1'
+
+  credentials:
+    access_key_id: 'access-key'
+    secret_access_key: 'secret-access-key'
+
+  # Maximum amount to upload in one request to object storage
+  max_upload_part: 2MB
```

Comment:
Is it a typo? Part must be at least 5 MB in size

Reply:
Oup's, should be 2GB